MediaRecorder.bs

```diff
@@ -575,9 +575,10 @@ The UA will throw a {{DOMException}} when the error can be detected at the time
 that the call is made. In all other cases the UA will <a>fire an event</a> named
 {{MediaRecorderErrorEvent}}.  If recording has been started and not yet stopped
 when the error occurs, let <var>blob</var> be the {{Blob}} of collected data so
-far; after raising the error, the UA will <a href="#to-fire-a-blob-event">fire a
-dataavailable event</a> with <var>blob</var>; immediately after the UA will then
-<a>fire an event</a> named <code>stop</code>.
+far; before raising the error, the UA will <a href="#to-fire-a-blob-event">fire a
```

Comment:
(After some offline chatting, we've decided that is best to beef up this subsection and make it normative with the appropriate writing, otherwise is quite misleading. New PS coming imminently.